firefox-ios/Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -502,18 +499,7 @@ class TabManagerImplementation: NSObject, TabManager, FeatureFlaggable {
         isRestoringTabs = true
         AppEventQueue.started(.tabRestoration(windowUUID))
 
-        guard tabMigration.shouldRunMigration else {
-            logger.log("Not running the migration",
-                       level: .debug,
-                       category: .tabs)
-            restoreOnly()
-            return
-        }
-
-        logger.log("Running the migration",
-                   level: .debug,
-                   category: .tabs)
-        migrateAndRestore()
+        restoreTabs()
```

Comment:
Renamed `restoreOnly()` to `restoreTabs()` since we don't have `migrateAndRestore()` anymore